docs/developer/experimental-repository.md

```diff
@@ -13,9 +13,9 @@ I will not hand-over the instructions to random newbies though!
 
 ## Consequences of using packages in experimental
 
-* Risk of downtimes, including long (bitcoin chain resync)
+* Risk of downtime, including long (bitcoin chain resync)
```

Comment:
There could be multiple downtimes. Not sure what's best stylistically.

Reply:
the plural form of `downtime` is also `downtime`.
https://www.wordhippo.com/what-is/the-plural-of/downtime.html

Reply:
Ah, TIL, OK.